app/src/screens/SearchModal.js

```diff
@@ -21,9 +23,18 @@ type Props = {
 export const SearchModal = (props: Props) => {
   const { navigation } = props;
   const title = toUpperCase(navigation.getParam('title'));
-  const category = navigation.getParam('category');
+  const type = navigation.getParam('category');
```

Comment:
I think I preferred the previous name category, either that or if the navigation param is also called type. To me it's confusing to call them two different things. 

```suggestion
  const category = navigation.getParam('category');
```

Reply:
Or 
```suggestion
  const sortingType = navigation.getParam('sortingType');
```

since "type" is such a general word.